R/tm_variable_browser.R

```diff
@@ -940,7 +940,8 @@ plot_var_summary <- function(var,
 
     if (numeric_as_factor) {
       var <- factor(var, levels = sort(unique(var)))
```

Comment:
I don't think the `sort(unique(` is necessary here.

Reply:
Levels will always be unique and levels being sorted is the default behavior. One runs `levels = unique(x)` to prevent them from being sorted.
```
x <- letters[4:1]
factor(x)
factor(x, levels = unique(x))
```

Reply:
It's not that big a deal though.

Reply:
Oh! I prefer just using unique in that case. Thanks for suggesting.

Reply:
But I think it's best to keep it this way:
```r
> var <- c(1, 3, 2, 1, 3, 1, 2, 1, 2)
> var <- factor(var, levels = sort(unique(var)))
> var
[1] 1 3 2 1 3 1 2 1 2
Levels: 1 2 3
> var <- c(1, 3, 2, 1, 3, 1, 2, 1, 2)
> var <- factor(var, levels = unique(var))
> var
[1] 1 3 2 1 3 1 2 1 2
Levels: 1 3 2
```

Reply:
Ah, but the sorting may makes sense in this case, I think. You represent a numeric as a factor and you draw a histogram.

Reply:
Okay, this is perfect! I agree that this is better.